### ISSUE TITLE
Visual Studio Code Windows version Issue

### ISSUE THREAD

**Issue by Rohitbiradar111** [microsoft/vscode#219860]
![Screenshot 2024-07-03 182725](https://github.com/microsoft/vscode/assets/114158641/d3e95977-5752-4a8d-84e1-023922e0a923)
Type: <b>Bug</b>

When i opened vs code, it says vs code on windows 8 will stop receiving updates,consider upgrading your windows version, but i am having windows 11. Then why i am i being shown this notification. I even checked for updates but it says no updates available. Also i am not able to sync the settings despite being login in. Please look into this matter.

VS Code version: Code 1.79.2 (695af097c7bd098fbf017ce3ac85e09bbc5dda06, 2023-06-14T08:57:04.379Z)
OS version: Windows_NT x64 6.2.9200
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i3-1215U (8 x 2496)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.68GB (2.00GB free)|
|Process Argv|--crash-reporter-id eb289106-11d2-40c7-8535-bb93afafe2ac|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
codesnap|adp|1.3.4
githistory|don|0.6.20
gitlens|eam|15.1.0
code-runner|for|0.12.2
git-graph|mhu|1.30.0
indent-rainbow|ode|8.3.1
java|red|1.32.0
errorlens|use|3.20.0
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.3.1
vscode-java-debug|vsc|0.58.0
vscode-java-dependency|vsc|0.23.7
vscode-java-pack|vsc|0.27.0
vscode-java-test|vsc|0.41.1
vscode-maven|vsc|0.44.0
vscode-icons|vsc|12.8.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyonecf:30548226
vscrpc:30673769
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
pythontbext0:30879054
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
chatpanelt:31048053
dsvsc021:30996838
0ee40948:31013168
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
2f103344:31071589
legacy_priority:31082724

```

</details>

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.90.2. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by Rohitbiradar111** [comment]
![Screenshot 2024-07-03 183508](https://github.com/microsoft/vscode/assets/114158641/6736b7d3-2d92-403f-8be1-ac4b5d3bd3df)

**Comment by gjsjohnmurray** [comment]
Have you configured code.exe (or the Windows shortcut that you launch it from) to use Windows 8 Compatibility mode?

**Comment by Rohitbiradar111** [comment]
> Have you configured code.exe (or the Windows shortcut that you launch it from) to use Windows 8 Compatibility mode?

No

**Comment by Rohitbiradar111** [comment]
The issue has been solved now.
Here's what I did: I uninstalled VS Code and then installed it again.
Note : I was still signed in.
After installing and opening Vs Code, it was running under administrator, i fixed this by going into the properties of Vs Code -> compatibility tab -> unchecked run this program as an administrator. 
Again when i opened VS Code, the terminal was blank; it was not responding. I searched the web and found out a solution.
Again in the properties of VS Code, -> compatibility tab -> unchecked the compatibility mode i.e run this program in the compatibility mode.
Right now everything works fine.
